src/Frontend/Modules/Blog/Engine/Model.php

```diff
@@ -762,6 +763,14 @@ private static function completeBlogPost(array $blogPost)
             }
         }
 
+        if (isset($blogPost['id'])) {
+            $blogPost['tags'] = FrontendTagsModel::getForItem(
+                'Blog',
```

Comment:
Is there no constant for this? Something like `Module::BLOG` or `self::MODULE_NAME` might be more maintainable?